plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogAccessOperations.java

```diff
@@ -497,8 +497,7 @@ public void testInformationSchemaColumns()
                             session,
                             "SELECT * FROM information_schema.columns WHERE table_schema = CURRENT_SCHEMA AND table_name LIKE 'test_select_i_s_columns%'",
                             ImmutableMultiset.<GlueMetastoreMethod>builder()
-                                    .addCopies(GET_TABLES, 7)
-                                    .addCopies(GET_TABLE, tables * 2)
+                                    .add(GET_TABLES)
```

Comment:
7x times fewer GET_TABLES calls.
Absolutely no GET_TABLE calls. I think it's worth a highlight.